src/extjson/models.rs

```diff
@@ -256,16 +255,13 @@ impl DateTime {
                 Ok(crate::DateTime::from_millis(date))
             }
             DateTimeBody::Relaxed(date) => {
-                let datetime: chrono::DateTime<Utc> =
-                    chrono::DateTime::parse_from_rfc3339(date.as_str())
-                        .map_err(|_| {
-                            extjson::de::Error::invalid_value(
-                                Unexpected::Str(date.as_str()),
-                                &"rfc3339 formatted utc datetime",
-                            )
-                        })?
-                        .into();
-                Ok(crate::DateTime::from_chrono(datetime))
+                let datetime = crate::DateTime::parse_rfc3339_str(date.as_str()).map_err(|_| {
```

Comment:
Here and elsewhere I changed it from the "parse into external datetime, convert to bson datetime" pattern to just directly using the methods `bson::DateTime` provides.